package.json

```diff
@@ -33,8 +33,9 @@
     "dependencies": {
         "@emotion/react": "^11.10.4",
         "@emotion/styled": "^11.10.4",
-        "@moderntribe/sitebuilder": "^1.1.0",
-        "@moderntribe/storebuilder": "^1.1.0",
+        "@moderntribe/wme-utils": "^0.1.0",
```

Comment:
Does this package need to define `wme-utils` as a dependency?

Reply:
Let me double-check.

Reply:
Thanks for the nudge on this. I've removed the package from the dependencies, as it was not necessary.